irispy/iris_tools.py

```diff
@@ -246,7 +246,7 @@ def get_iris_response(pre_launch=False, response_file=None, response_version=Non
 
 
 @custom_model
-def _gaussian1d_on_linear_bg(x, amplitude=None, mean=None, standard_deviation=None,
+def _gaussian1d_on_linear_bg(x=None, amplitude=None, mean=None, standard_deviation=None,
```

Comment:
I believe for @custom_model, ```x``` should be a positional argument, not a kwarg.  So the original version is actually correct.